packages/console/src/App.tsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { BrowserRouter, useLocation, useNavigate } from 'react-router-dom';
 
 import './scss/normalized.scss';
 import * as styles from './App.module.scss';
 import Content from './components/Content';
 import Sidebar from './components/Sidebar';
+import { sections } from './components/Sidebar/consts';
+import { getPath } from './components/Sidebar/utils';
```

Comment:
nit: would it be handier to make them all exportable through Sidebar Components
import. Sidebar, { sections, getPath } from './components/Sidebar`?

Reply:
cool